2023-10-18T00:22:47.5885475Z ##[group]Run actions/setup-python@v4
2023-10-18T00:22:47.5886035Z with:
2023-10-18T00:22:47.5886403Z   python-version: 3.8
2023-10-18T00:22:47.5886863Z   architecture: x64
2023-10-18T00:22:47.5887271Z   cache: pip
2023-10-18T00:22:47.5887640Z   check-latest: false
2023-10-18T00:22:47.5888274Z   token: ***
2023-10-18T00:22:47.5888678Z   update-environment: true
2023-10-18T00:22:47.5889180Z   allow-prereleases: false
2023-10-18T00:22:47.5889632Z ##[endgroup]
2023-10-18T00:22:47.7921798Z ##[group]Installed versions
2023-10-18T00:22:47.8011947Z Successfully set up CPython (3.8.18)
2023-10-18T00:22:47.8013225Z ##[endgroup]
2023-10-18T00:22:48.8329016Z [command]/opt/hostedtoolcache/Python/3.8.18/x64/bin/pip cache dir
2023-10-18T00:22:58.4335329Z /home/runner/.cache/pip
2023-10-18T00:22:58.9133724Z Received 6245959 of 6245959 (100.0%), 34.4 MBs/sec
2023-10-18T00:22:58.9134894Z Cache Size: ~6 MB (6245959 B)
2023-10-18T00:22:58.9169439Z [command]/usr/bin/tar -xf /home/runner/work/_temp/469f1cd5-1e57-4faa-8c67-faad626f5d03/cache.tzst -P -C /home/runner/work/GitHub_Stats_A/GitHub_Stats_A --use-compress-program unzstd
2023-10-18T00:22:58.9557763Z Cache restored successfully
2023-10-18T00:22:58.9578707Z Cache restored from key: setup-python-Linux-22.04-Ubuntu-python-3.8.18-pip-60f2a6c0212a9b74a155cd81b58dd07fd8b180833132566260d95f4de42fe218
